helloworld-mutual-ssl-secured/configure-client-truststore.cli

```diff
@@ -0,0 +1,11 @@
+#extract the certificate for the server keystore into a file called serverCert.crt
+/subsystem=elytron/key-store=applicationKS:export-certificate(alias=server, path=serverCert.crt, relative-to=jboss.server.config.dir, pem=true)
+
+# create a truststore for the client called client.truststore and add it to the server config directory
+/subsystem=elytron/key-store=clientTS:add(path=client.truststore, relative-to=jboss.server.config.dir, credential-reference={clear-text=secret}, type=PKCS12)
+
+# import that certificate into the client truststore
```

Comment:
Same question here, since this is for the client side, why is this needed?

Reply:
Maybe this file could just be renamed to export-server-certificate.cli and it would just export the certificate to a file.

Reply:
Wouldn't configure-client-truststore.cli make more sense, since this is what the script is meant to do? 
The purpose of this script is to extract the server's certificate into the client's truststore to set up the client programmatically during tests.

Reply:
Since the truststore is only used for the test class, I think it would make sense to only create the truststore programmatically in the test itself.